src/core/tokens/_core-shadows.scss

```diff
@@ -0,0 +1,10 @@
+// CORE TOKENS: SHADOWS
+// This partial defines global shadow related tokens
+
+$shadow-size-small: 0px 4px 8px;
+$shadow-size-medium: 0px 6px 20px;
+$shadow-size-large: 0px 15px 50px;
+
+$shadow-small-opacity: $opacity-20;
+$shadow-medium-opacity: $shadow-small-opacity;
```

Comment:
I remember from the previous pr that this is intentional right?

Reply:
yes. but just for readbility.